src/python/pants/pantsd/service/scheduler_service.py

```diff
@@ -14,6 +14,7 @@
 from future.utils import PY3
 from twitter.common.dirutil import Fileset
 
+from pants.engine.fs import PathGlobs, PathGlobsAndRoot, Snapshot
```

Comment:
`PathGlobsAndRoot` is unused and causing the lint failure, remove.

Reply:
removed PathGlobsAndRoot